obs-packaging/linux-container/kata-linux-container.spec-template

```diff
@@ -4,17 +4,16 @@
 # This specialization allows us to optimize memory footprint and boot time.
 #
 
-%define bzimage_arch x86
-
 Name:           kata-linux-container
 Version:        @VERSION@.@CONFIG_VERSION@
 Release:        @RELEASE@.<B_CNT>
 License:        GPL-2.0
 Summary:        The Linux kernel optimized for running inside a container
+Group:          System/Kernel
 Url:            http://www.kernel.org/
-Group:          kernel
 Source0:        https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-@VERSION@.tar.xz
-Source1:        config
+Source1:        kata-configs.tar.gz
```

Comment:
- `kata-kernel-config` (or `kata-kernel-configs`) would be clearer I think.
- How about adding a macro for this name as it's used in the `%setup` below?

Reply:
- Ok for renaming the tarball / folder.
- Updated the debian.rules to refer to this using a variable
- About the spec file for rpm, I'd prefer to keep the `Source:` as it is now. I find it easier to read/document what are the package source.